### ISSUE TITLE
humanSeg for iOS

### ISSUE THREAD

**Issue by zhangjunhst** [PaddlePaddle/PaddleSeg#3450]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

iOS端 使用[paddle-lite](https://github.com/PaddlePaddle/Paddle-Lite-Demo/blob/develop/human_segmentation/ios)代码替换模型后返回的数据与效果不符合，相同的模型安卓根据 https://github.com/PaddlePaddle/PaddleSeg/issues/279#issuecomment-638821411 修改后返回正确，而iOS不知道改如何做对应的改造

**Comment by zhangjunhst** [comment]
Model: DeepLabv3p
Transforms:
- ResizeByLong: {long_size: 512}
- Padding:
    im_padding_value: [127.5, 127.5, 127.5]
    label_padding_value: 255
    target_size: 512
- Normalize:
    max_val: [255.0, 255.0, 255.0]
    mean: [0.5, 0.5, 0.5]
    min_val: [0, 0, 0]
    std: [0.5, 0.5, 0.5]
TransformsMode: BGR
_Attributes:
  eval_metrics: {miou: 0.6704429236012107}
  fixed_input_shape: null
  labels: [background, pointer, scale]
  model_type: segmenter
  num_classes: 3
_ModelInputsOutputs:
  test_inputs:
  - [image, image]
  test_outputs:
  - [pred, unsqueeze2_0.tmp_0]
  - [logit, softmax_0.tmp_0]
_init_params: {aspp_with_sep_conv: true, backbone: MobileNetV3_large_x1_0_ssld, class_weight: null,
  decoder_use_sep_conv: true, enable_decoder: true, encoder_with_aspp: true, ignore_index: 255,
  input_channel: 3, num_classes: 3, output_stride: 16, pooling_crop_size: null, use_bce_loss: false,
  use_dice_loss: false}
completed_epochs: 0
status: Infer
version: 1.2.8
 
模型的参数

**Comment by TingquanGao** [comment]
Thanks for this issue. As it has been inactive for a long time, we would close it. If you has any questions, please feel free to reopen or new issue, and we will follow up and resolve it.
